operations/mimir-tests/test-ruler-query-path.jsonnet

```diff
@@ -0,0 +1,24 @@
+local mimir = import 'mimir/mimir.libsonnet';
```

Comment:
If you agree on renaming `operations/mimir/ruler-query-path.libsonnet`, then I would rename this file accordingly.